fluentd/include/Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'elasticsearch-xpack', '~>7.9'
 gem 'fluent-plugin-concat', '~>2.5'
 gem 'fluent-plugin-detect-exceptions', '~>0.0'
 gem 'fluent-plugin-elasticsearch', '~>4.3'
+gem "fluent-plugin-cloudwatch-logs", "~> 0.13.4"
```

Comment:
Since you're updating the Gemfile, please run `make update-fluentd-gemfile.lock` and commit those changes.